Sources/Workspace/InitPackage.swift

```diff
@@ -332,24 +374,36 @@ public final class InitPackage {
 
                             // A client of the library, which is able to use the macro in its own code.
                             .executableTarget(name: "\(pkgname)Client", dependencies: ["\(pkgname)"]),
-
-                            // A test target used to develop the macro implementation.
-                            .testTarget(
-                                name: "\(pkgname)Tests",
-                                dependencies: [
-                                    "\(pkgname)Macros",
-                                    .product(name: "SwiftSyntaxMacrosTestSupport", package: "swift-syntax"),
-                                ]
-                            ),
+                    \(testTarget)
```

Comment:
indentation looks off here, unless it's a diff rendering issue specific to GitHub itself?

Reply:
The indentation is correct—because `testTarget` is a multi-line string, it already has its own indentation applied. If we applied indentation here, it would only affect the first line.

(There's an argument to be made for running it all through SwiftSyntax and calling `formatted()` but that's not a change appropriate for this PR IMHO.)